packages/@aws-cdk/cloudformation-include/test/test-templates/yaml/short-get-att.yaml

```diff
@@ -0,0 +1,32 @@
+Resources:
+  Bucket1:
+    Type: AWS::S3::Bucket
+    Properties:
+      BucketName: SomeBucketName
+      ReplicationConfiguration:
+        Role: "some-role"
+        Rules:
+        - Destination:
+            Bucket: Bucket1
+            StorageClass: STANDARD
+          Status: Enabled
+  Bucket2:
+    Type: AWS::S3::Bucket
+    Properties:
+      BucketName: 
+        !GetAtt [ Bucket1, BucketName ]
+  Bucket3:
+    Type: AWS::S3::Bucket
+    Properties:
+      BucketName:
+        !GetAtt Bucket1.BucketName
+  Bucket4:
+    Type: AWS::S3::Bucket
+    Properties:
+      BucketName:
+        !GetAtt Bucket1.Rules.Destination
```

Comment:
Is this valid CFN? I thought GetAtt took a list of2 strings as arguments?

Reply:
This doc https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-getatt.html shows this example: 

`!GetAtt myELB.DNSName`

the docs might be lying though, I'll test it.

Reply:
Right. But your test is `Bucket1.Rules.Destination` (you have 2 `.` in your expression that you pass to `Fn::GetAtt`).

Reply:
yes, I believe that's valid, the docs show this:

`SourceSecurityGroupOwnerId: !GetAtt myELB.SourceSecurityGroup.OwnerAlias`

Reply:
OK. I do see that a LoadBalancer has an output attribute `SourceSecurityGroup.GroupName`: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-elb.html#aws-properties-ec2-elb-return-values-fn--getatt

But there is not output attribute called `Rules.Destination`for `AWS::S3::Bucket`: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-s3-bucket.html#aws-properties-s3-bucket-return-values-fn--getatt